backend/src/apps/owasp/utils/certificate_provider.py

```
@@ -0,0 +1,74 @@
+"""Certificate provider classes for Contributor Recognition Program."""
```

Ruff (0.15.15)

[warning] File `backend/apps/owasp/services/certificate_providers.py` is part of an implicit namespace package. Add an `__init__.py`.

(INP001)